### PR TITLE
Language section added to home page

### PR DESCRIPTION
**Pull request by YogeshSharma01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel Please review.

File: index.html [hunk 1]
```diff
@@ -262,7 +319,7 @@ <h2 class="recent-contributors-head">Recent Contributors</h2>
         </div>
       </div>
 
-      <h2 class="contributors-head">Contributors</h2>
+      <h2 class="contributors-head" id="contributer">Contributors</h2>
```
Comment (govindgoel) [inline index.html:322]: typo in the div name @YogeshSharma01

Comment (YogeshSharma01) [reply index.html]: Yeah Thanks, @govindgoel I have fixed the typo

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel I have squashed the commits. Please merge the PR.

**Comment by jywarren** [comment]
Hi, thanks for your help! can you connect us to the issue this is addressing? And, would you be able to share a screenshot? Thanks a lot!!!

**Comment by govindgoel** [comment]
@jywarren  closes #337 This is an enhancement feature, I think this will surely help.

**Comment by YogeshSharma01** [comment]
Hi, @jywarren  this is an enhancement feature as @govindgoel  mentioned. 

https://user-images.githubusercontent.com/51679200/106174852-7387f280-61bb-11eb-9de3-0348c97b3d0f.mov

**Comment by YogeshSharma01** [comment]
https://user-images.githubusercontent.com/51679200/109194543-89d79d00-77bf-11eb-907e-f126c6d415d0.mov

File: index.html [hunk 2]
```diff
@@ -21,6 +21,9 @@
     <link rel="manifest" href="./manifest.json">
     <script src="./src/scripts/pwainit.js"></script>
 
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (daemon1024) [inline index.html:25]: Why was it needed to fetch from CDN? Don't we already have it as a dependency?

Comment (YogeshSharma01) [reply index.html]: I have removed
Thanks

File: examples/demo.js [hunk 3]
```diff
@@ -11,6 +11,24 @@ document.addEventListener('DOMContentLoaded', function () {
...
+
+    request.open('GET', 'https://api.github.com/users/publiclab/repos', true);
+
+    request.onload = function () {
+    var data = JSON.parse(this.response);
+    var statusHTML = '';
+    $.each(data, function (i, status) {
+        statusHTML += '<div class="conatiner-language">';
+        statusHTML += '<div class="repo-name"><a href="' + status.html_url + '">' + status.name + '</a></div>';
+        statusHTML += '<divs class="lang-name">' + status.language + '</div>';
```
Comment (jywarren) [inline examples/demo.js:25]: This is looking very good - however, could we use standard Bootstrap badges for these languages?  https://getbootstrap.com/docs/4.4/components/badge/

Also, i think `divs` is a typo!

Thanks for your work on this! If there's a way to ensure languages are a little closer associated with the project names, that's probably good too, as currently it seems like maybe it's easy to confuse which language they correspond to. What if we did it like:

```
Project Name
[language]
```

**Comment by YogeshSharma01** [comment]
Hi, @jywarren we are using Bootstrap 3.4.0 in our project which uses only a default color which is grey. The changes I've done are with this version only.
If we need to change its color, then either we upgrade to 4.0 or manually set it.
Though I think that this color is suitable as per the website. What do you say?

https://user-images.githubusercontent.com/51679200/109634134-e4317e80-7b6e-11eb-88c9-e6363f84920e.mov

**Comment by jywarren** [comment]
I think this looks great. The only and hopefully final change I'd ask is that we filter out `null` values, is that ok? Thank you so much!

File: index.html [hunk 4]
```diff
@@ -191,6 +245,14 @@ <h3>3.<br />Create a welcoming issue for someone else</h3>
       </div>
 
       <br />
+      <!-- Langauge section -->
+      <h2><span id="fto-issue">Languages</span> Uses</h2>
```
Comment (jywarren) [inline index.html:249]:
```suggestion
      <h2><span id="fto-issue">Languages</span> Used</h2>
```

Comment (YogeshSharma01) [reply index.html:249]: ok

**Comment by YogeshSharma01** [comment]
Hi, @jywarren, in case if null, should we not add a label to the repo or some text instead? Like - NA

**Comment by jywarren** [comment]
Sure, but let's not use a label - just plain text like "N/A" is fine!
Thanks!

**Comment by YogeshSharma01** [comment]
Hi, @jywarren all set, and thanks 

https://user-images.githubusercontent.com/51679200/110002700-b483a180-7d3b-11eb-9f2c-b63e71af74af.mov

Please Review

**Comment by jywarren** [comment]
Oh, I'm sorry -- would it also be possible to move the listings to the bottom of the page? Perhaps just above "Stale Issues" or below Contributors? Thank you, I'm sorry i thought I'd made that suggestion already. Many thanks!

**Comment by YogeshSharma01** [comment]
No problem sir, don't be sorry I'll set this section above the stale issues
cause it would looks good at that place only as at the bottom we have
dependency chart so I think we can also set it between the contributors and
the dependency chart what do you think?
Warm Regards,
Yogesh Sharma
Thanks you

**Comment by YogeshSharma01** [comment]
I think it should be fine between contributors and dependency chart section

**Comment by YogeshSharma01** [comment]
Hi, @jywarren  I have set the language section just below the contributor's section. 

https://user-images.githubusercontent.com/51679200/110231922-963ac300-7f40-11eb-9b0f-1fdc166fad73.mov

**Comment by jywarren** [comment]
Hi! Great. Restarting the checks to try to get gitpod to pass. Thanks!!

**Comment by YogeshSharma01** [comment]
> Hi! Great. Restarting the checks to try to get gitpod to pass. Thanks!!

Yeah Sure.

**Comment by jywarren** [comment]
Hm, i'm so sorry, this is seeming to get stuck. Would you mind rebasing it and/or adding one more commit to see if we can get GitPod to go? Otherwise I can turn off the GitPod requirement as maybe I have it misconfigured. Apologies for the trouble!

**Comment by govindgoel** [comment]
@YogeshSharma01 it's strange that gitpod is geeting stuck can you try once with squashing all the commits then run it again.

**Comment by YogeshSharma01** [comment]
I don't think squashing all the commits would work @jywarren what do you think? should I squashing all the commits and try again?

**Comment by jywarren** [comment]
Oh that's OK - i'll just turn off the gitpod check. Thanks!!!

**Comment by jywarren** [comment]
Great work! Awesome to finally see this wrapped up! Thanks for your patience and persistence!!!

**Pull request merged**